components/atoms/ActionButton.js

```diff
@@ -0,0 +1,192 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import { useEffect } from 'react'
+
+/**
+ * Button component
+ */
+export function ActionButton(props) {
+  //Styling for buttons and links
+  const basicStyle =
+    'rounded-sm focus:ring-1 focus:ring-black focus:ring-offset-2'
+  const defaultStyle =
+    'py-2 px-4 bg-custom-blue-blue text-white border border-custom-blue-blue active:bg-custom-blue-dark hover:bg-custom-blue-light'
+  const secondaryStyle =
+    'py-2 px-4 bg-white text-custom-blue-blue border border-custom-blue-blue active:bg-gray-400 hover:bg-gray-200'
+  const tertiaryStyle =
+    'py-2 underline hover:text-canada-footer-hover-font-blue text-canada-footer-font'
+  const disabledStyle =
+    'py-2 px-4 bg-gray-light text-gray-600 border border-gray-md'
+
+  //Activate Links with spacebar
+  useEffect(() => {
+    let link = document.getElementById(props.id)
+    if (link) {
+      link.addEventListener('keydown', (event) => {
+        if (event.key === 'Spacebar' || event.key === ' ') {
+          event.preventDefault()
+          link.click()
+        }
+      })
+    }
+  })
+
+  return props.href ? (
+    <Link href={props.href}>
+      <a
+        className={`${basicStyle}
+        ${
+          !props.secondary &&
+          !props.tertiary &&
+          !props.disabled &&
+          !props.custom
+            ? defaultStyle
+            : props.className
+        }
+        ${props.secondary && !props.disabled ? secondaryStyle : props.className}
+        ${props.tertiary && !props.disabled ? tertiaryStyle : props.className}
+        ${props.custom && !props.tertiary ? props.custom : ''}
+        ${props.disabled ? disabledStyle : props.className}`}
+        onClick={props.onClick}
+        id={props.id}
+        data-testid={props.dataTestId}
+        data-cy={props.dataCy || props.id}
+        data-cy-button={props.dataCyButton}
```

Comment:
Maybe remove the data fields for now, as they won't be used for a while I think.

Reply:
I would personally leave them because we should use them. even if its in the future.

But thats just me

Reply:
I vote to leave them as there is a lot of functionality in this button that will become useful. maybe some comments might be nice thought to explain what different parts do or are for etc.

Reply:
That makes sense, we could also add that they aren't doing anything atm so we ignore it for now if we add more to it

Reply:
ActionButton is not used yet on any page, we should wait until we decide on the colours to start deleting code that we might need later